src/ui/alert/Alert.tsx

```diff
@@ -1,41 +1,42 @@
-import React, { useCallback, memo } from 'react';
+import React, { useCallback, memo, useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
 
-import CloseIcon from '@material-ui/icons/Close';
-
-import { usePortal } from 'utils';
+import Snackbar from '@material-ui/core/Snackbar';
 
 import csx from './Alert.scss';
 
 namespace Alert {
-  export namespace Events {
-    export type Close = React.MouseEvent<SVGSVGElement, MouseEvent>;
-  }
-
   export type Types = 'warning' | 'error' | 'success' | 'info';
 
-  export type OnClose = (idx: number) => void;
-
   export interface Props {
-    idx: number;
     message: string;
     type?: Types;
-    onClose: OnClose;
+    onClose: () => any;
   }
 }
 
 const Alert = memo(
-  ({ idx, message, type = 'error', onClose }: Alert.Props) => {
-    const render = usePortal();
+  ({ message, type = 'error', onClose }: Alert.Props) => {
+    const [isOpen, setIsOpen] = useState(true);
+    const location = useLocation();
+
+    useEffect(() => {
```

Comment:
This is not a bug when you change route and alert stay. Imagine example if you will be on templates search and scroll down - if an error occurs and you scroll - the error instantly disappeared. Please remove the code connected to hide alert.